<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1416</width>
    <height>822</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>0</y>
      <width>1318</width>
      <height>620</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QFrame" name="frame">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="minimumSize">
        <size>
         <width>1000</width>
         <height>618</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QFrame" name="frame_2">
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>618</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>200</width>
         <height>618</height>
        </size>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <widget class="QPushButton" name="pushButton">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>40</y>
          <width>121</width>
          <height>61</height>
         </rect>
        </property>
        <property name="text">
         <string>图片识别</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_2">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>160</y>
          <width>121</width>
          <height>61</height>
         </rect>
        </property>
        <property name="text">
         <string>视频识别</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_3">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>280</y>
          <width>121</width>
          <height>61</height>
         </rect>
        </property>
        <property name="text">
         <string>实时识别</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_4">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>400</y>
          <width>121</width>
          <height>61</height>
         </rect>
        </property>
        <property name="text">
         <string>退出系统</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_5">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>520</y>
          <width>121</width>
          <height>61</height>
         </rect>
        </property>
        <property name="text">
         <string>联系我们</string>
        </property>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1416</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_4</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1288</x>
     <y>434</y>
    </hint>
    <hint type="destinationlabel">
     <x>1235</x>
     <y>695</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
